clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessor.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 Lumberjack authors (see AUTHORS file)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.abcxyz.lumberjack.auditlogclient.processor;
+
+import com.abcxyz.jvs.JvsClient;
+import com.abcxyz.lumberjack.v1alpha1.AuditLogRequest;
+import com.google.api.client.util.StringUtils;
+import com.google.cloud.audit.AuditLog;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import org.apache.commons.codec.binary.Base64;
+import com.google.inject.Inject;
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor(onConstructor = @__({@Inject}))
+public class JustificationProcessor {
+  private static final String JUSTIFICATION_LOG_METADATA_KEY = "justification";
```

Comment:
it is justification_token in golang, is it not? https://github.com/abcxyz/lumberjack/commit/a76a124e56323b48128f804199d9c0903dc9abf4#diff-60c8ed893b3d6f4959e9200c588be5b2df2ae81255d006f0b17e718e307232cdR46

Reply:
I changed it per https://github.com/abcxyz/lumberjack/pull/272. See reason in that PR's description.